gentest/fixtures/YGPercentageTest.html

```diff
@@ -0,0 +1,86 @@
+<div id="percentage_width_height" experiments="Rounding" style="width: 200px; height: 200px; flex-direction: row;">
```

Comment:
there is no need to write tests for border, as border can't have percentage in css.